src/styles/main.scss

```diff
@@ -0,0 +1,18 @@
+.catalog {
+  display: grid;
+  grid-template-columns: 1fr;
+  gap: $gap-vertical;
+  padding: $paddings-vertical $paddings-horizontal;
```

Comment:
Ensure that the variables `$gap-vertical`, `$paddings-vertical`, and `$paddings-horizontal` are defined in the 'variables.scss' file. Missing variable definitions will lead to compilation errors.